dashboard/package.json

```diff
@@ -43,7 +43,6 @@
     "typescript": "4.9.5",
     "unique-names-generator": "^4.7.1",
     "universal-cookie": "^4.0.4",
-    "uuid": "^9.0.0",
```

Comment:
Did you re-run `yarn` after this? I'm surprised the `yarn.lock` file was not touched.

Reply:
I did and it didn't regenerate it. I was also surprised